.tools/test/eventbridge_rule_with_sns_fanout/producer_stack/producer_stack.py

```diff
@@ -14,8 +14,8 @@
 class ProducerStack(Stack):
     def __init__(self, scope: Construct, id: str, **kwargs) -> None:
         super().__init__(scope, id, **kwargs)
-        acct_config = self.get_yaml_config("../../config/targets.yaml")
-        resource_config = self.get_yaml_config("../../config/resources.yaml")
+        acct_config = self.get_yaml_config("../config/targets.yaml")
```

Comment:
What is `..` going to be relevant to? Probably better to replace this with `Path(__file__).parent / '..' / 'config' / 'targets.yaml'` or similar

Reply:
It's relative to the directory where the CDK stack is designed to be run. It won't work from any other directory. In a use case where the file could be executed directly from any location, this would be an improvement. In the use case where the file is invoked indirectly from a fixed location within a rigid third party framework, relative pathing like this is fine.

Is there any other reason you think this would be an improvement? Open to other reasoning.

Reply:
Well, I am a CDK user, and I use other things, and I _thought_ that was the reason but I wasn't entirely certain. So no strong technical reasons one or the other, but it did take me a minute to puzzle out & be certain. And I think I've written basically what you wrote when I got asked what a relative path meant in a different CDK script.